123_state_tasks.R

```diff
@@ -1,18 +1,37 @@
-do_state_tasks <- function(oldest_active_sites) {
+do_state_tasks <- function(oldest_active_sites, ...) {
 
   # Define task table rows
-  # TODO: DEFINE A VECTOR OF TASK NAMES HERE
+  state_codes = oldest_active_sites$state_cd
 
   # Define task table columns
   download_step <- create_task_step(
-    step_name = 'download'
-    # TODO: Make target names like "WI_data"
-    # TODO: Make commands that call get_site_data()
+    step_name = 'download',
+    target_name = function(task_name, step_name, ...){
+        sprintf('%s_data', task_name)
+    },
+    #command = 'get_site_data(oldest_active_sites, target_name, parameter)'
+    command = function(task_name, ...){
+        sprintf("get_site_data(oldest_active_sites, I('%s'), parameter)", task_name)
+    }
   )
 
-  # Return test results to the parent remake file
-  return(list(
-    example_target_name = download_step$target_name(task_name='WI', step_name='download'),
-    example_command = download_step$command(task_name='WI', step_name='download')
-  ))
+  # create the task plan
+  task_plan <- create_task_plan(
+    task_names = state_codes,
+    task_steps = list(download_step),
+    add_complete = FALSE)
+
+  create_task_makefile(
+    task_plan,
+    '123_state_tasks.yml',
```

Comment:
picky, but might want to name the arguments for task_makefile here (and `task_plan`?)